kola-denylist.yaml

```diff
@@ -100,3 +100,15 @@
   osversion:
     - centos-9
     - centos-10
+
+- pattern: ostree.remote
+  tracker: https://github.com/coreos/rhel-coreos-config/issues/34
+  snooze: 2025-07-05
+  arches:
+    - x86_64
```

Comment:
I think this tests also runs on multi-arch and I _think_ it uses the fedora infra too.  We should probably snooze this across the board.